projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -417,7 +417,7 @@
 	border-color: var(--jetpack--contact-form--border-color);
 	border-radius: var(--jetpack--contact-form--border-radius);
 	border-style: var(--jetpack--contact-form--border-style);
-	border-width: var(--jetpack--contact-form--border-size);
+	border-width:  var(--jetpack--contact-form--border-left-size, var(--jetpack--contact-form--border-size));
```

Comment:
We should probably break these out to something like:
```css
border-top-width: var(--jetpack--contact-form--border-top-size, var(--jetpack--contact-form--border-size));
border-right-width: var(--jetpack--contact-form--border-right-size, var(--jetpack--contact-form--border-size));
border-bottom-width: var(--jetpack--contact-form--border-bottom-size, var(--jetpack--contact-form--border-size));
border-left-width: var(--jetpack--contact-form--border-left-size, var(--jetpack--contact-form--border-size));
```

Same in a few other cases in this file.

In reality I think they get overridden by inline styles, but I think it might still be a good idea, in case that changes in the future.